apps/src/code-studio/projectRedux.js

```diff
@@ -0,0 +1,125 @@
+/** Redux actions and reducer for the inidividual projects */
```

Comment:
wording nit: "for an individual project" might be clearer here

Reply:
Thanks! Will do.